proto/protovalidate/buf/validate/validate.proto

```diff
@@ -415,11 +409,6 @@ enum Ignore {
   // }
   // ```
   IGNORE_ALWAYS = 3;
-
-  // Deprecated: Use IGNORE_IF_UNPOPULATED instead. TODO: Remove this value pre-v1.
-  IGNORE_EMPTY = 1 [deprecated = true];
-  // Deprecated: Use IGNORE_IF_DEFAULT_VALUE. TODO: Remove this value pre-v1.
-  IGNORE_DEFAULT = 2 [deprecated = true];
```

Comment:
Is the reserved keyword on enums?

Reply:
Apparently, yes, reserved the old enum names as well.